web/readme.md

```diff
@@ -0,0 +1,9 @@
+you have to install npm and node in your computer.
```

Comment:
Missing Apache License header. And please add more descriptions and details about requisitions like npm.